airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -54,17 +54,19 @@ class AbstractSource(Source, ABC):
     @abstractmethod
     def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) -> Tuple[bool, Optional[Any]]:
         """
-        :param config: The user-provided configuration as specified by the source's spec. This usually contains information required to check connection e.g. tokens, secrets and keys etc.
-        :return: A tuple of (boolean, error). If boolean is true, then the connection check is successful and we can connect to the underlying data
-        source using the provided configuration.
-        Otherwise, the input config cannot be used to connect to the underlying data source, and the "error" object should describe what went wrong.
-        The error object will be cast to string to display the problem to the user.
+        :param config: The user-provided configuration as specified by the source's spec.
```

Comment:
is there a reason all this whitespace is changing?

Reply:
I think I just formatted this so I could read it better when I was starting off